src/main/resources/files/readme.md.secondary.tpl

```diff
@@ -36,6 +36,10 @@ However, any method to send a REST request with an appropriate header will work
 
 
 [# th:if="${mp_rest_client}"]
+### Demo Execution
+
+Once the service-a server is running (see `demo/service-a/readme.md`) and the service-b server is running, open your browser at the following URL to launch the test page and execute both projects: http://localhost:9080/index.html
```

Comment:
The port is not always `9080`. This readme is for service-b. I think it is a bit irrelevant to talk about the caller service-a.

Reply:
I suggest to remove this.

Reply:
The point was to make it clear that the servers for both service-a and service-b need be running before hitting index.html.

Reply:
I will change it to http://localhost:[# th:text="${port_service_a}"/]/index.html, but I think the comment about service-a is relevant in the JAXRS case.